### ISSUE TITLE
Prevent the usage of experimental API's

### ISSUE THREAD

**Issue by amilajack** [amilajack/eslint-plugin-compat#61]
Detect usage of API's whose specs have not been standardized

Idea:

```js
const buffer = new SharedArrayBuffer(1024);
// ^ Error: compat/no-experimantal-apis: Unallowed use of experimental APIs
```

**Comment by ljharb** [comment]
(unrelated to the concept, which is good - but SharedArrayBuffer is stage 4, and as such is fully standardized and officially in the language)

**Comment by amilajack** [comment]
@ljharb my bad. Edited my example to reflect your feedback
